### PR TITLE
Homepage branch for positioning and floating project - project 1

### PR DESCRIPTION
**Pull request by MaiRizk**
This PR contains the following;

- Structuring the whole article using semantic HTML
- Styling the whole article -all requirements have been added to the style-
- Adding README file with the requested template
- Adding linter to the project

### CODE REVIEW

**Comment by mricanho** [comment]
# 🔨 Invalidation error 🔨   

Hello @MaiRizk, this invalidation was a mistake and the request will be reviewed as soon as possible.

sorry


### 💻 👩🏽‍💻  Happy Coding!👨🏽‍💻  🖥️

File: styles.css [hunk 1]
```diff
@@ -0,0 +1,989 @@
...
+  color: rgb(85, 83, 83);
+}
+
+.more-a:hover {
+  text-decoration: underline;
+}
+
+.more-section-card a:hover {
+  color: rgb(85, 83, 83);
+}
```
Comment (Dipeshtwis) [inline styles.css]: the ` color: rgb(85, 83, 83)` used in different class. Use DRY principle.

e.g. - 
```
.more-section-card a:hover,
.bottom-editors-card a:hover,
.most-pop-link a:hover
{
  color: rgb(85, 83, 83);
}
```

**Review by Dipeshtwis — CHANGES_REQUESTED** [review]
# Status: ♻️ Changes Required ♻️
## Hi @MaiRizk , I'm Dipesh your reviewer for this PR 👩‍🚀 
This setup looks nice and clean. You have also used semantic HTML.
Also, I have to appreciate that your website looks similar to the original one and you have done a very good job so far. But in order to approve we have to work on some of the changes mentioned below.

## For Approval
- [ ] The image is not showing up in some of the sections I have mentioned below. I noticed it's because of the wrong path using `./Screenshot/3.png` instead of `/Screenshot/3.png`.

**Theory of inflation**

Screenshot (1279)

**Suggested newsletter for you**

Screenshot (1280)

- [ ] use the DRY principle (in CSS). I have mentioned this in the comment.

## [Optional]
- [ ] It's better to fix the footer (it sticks to the bottom part of the screen) ([link to page](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html?_r=0)).
---------

## Reminder

**"As described in the [Code reviews limit Policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) now, you have 2 code reviews left for this project."**

Code review by **Dipesh Kumar**

(Feel free to contact me on slack by the handle of  **deep** if you have any additional questions).
Happy Coding
[Dipesh Kumar](https://deeptwist.tk)
[Twitter](https://twitter.com/97deepeshkumar)

**Review by Dipeshtwis — APPROVED** [review]
# Status: Approved ✔️
## Hi @MaiRizk , I'm Dipesh your reviewer for this PR 👩‍🚀 

Good job on implementing the changes as requested in the previous review. Well done.
**You can now merge and submit.**


## [Optional]
- [ ] It's better to remove the border from the navbar button.
---------

Code review by **Dipesh Kumar**

(Feel free to contact me on slack by the handle of  **deep** if you have any additional questions).
Happy Coding
[Dipesh Kumar](https://deeptwist.tk)
[Twitter](https://twitter.com/97deepeshkumar)

**Pull request merged**